src/main/java/org/fcrepo/spec/testsuite/test/Container.java

```diff
@@ -18,13 +18,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.ibr.fedora.testsuite;
+package org.fcrepo.spec.testsuite.test;
```

Comment:
What is the purpose of the additional `test` package?

Reply:
The root path was com.ibr.fedora.  All the tests were in  com.ibr.fedora.testsuite
Thus com.ibr.fedora became org.fcrepo.spec.testsuite and com.ibr.fedora.testsuite became org.fcrepo.spec.testsuite.test  since the since the "testsuite" dir contains all project files whereas the "test" directory is the location of all the tests.